Graphics/Blank/JavaScript.hs

```diff
@@ -624,7 +618,17 @@ jsEscapeChar '\t' = "\\t"
 jsEscapeChar '\v' = "\\v"
 jsEscapeChar '\"' = "\\\""
 jsEscapeChar '\'' = "\\'"
```

Comment:
Another special case that exists in this version of the function, but not the GHCJS version. Curious.

Reply:
I think here you make sure that `'` works when string literal is enclosed in single quotes (`'I\'m single'`), whereas in GHCJS they probably rely on literals always being in double quotes (`"I'm double"`). But I am not 100% sure. In any case this seems harmless to have.

Reply:
Ah, I had overlooked the fact that GHCJS is only using `encodeJsonChar` within things that are double-quoted. And since [this site](https://mathiasbynens.be/notes/javascript-escapes) specifically mentions `\'` as a special character, I agree that it makes sense to include it here.